clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -1412,6 +1412,9 @@ def FunctionMultiVersioning
 
 def NoDeref : DiagGroup<"noderef">;
 
+// Bounds safety specific warnings
+def IgnoredBoundsSafety : DiagGroup<"ignored-bounds-safety">;
```

Comment:
This is `-Wignored-bounds-safety` is not used.

Reply:
Removed it!